src/dparse/ast.d

```diff
@@ -487,6 +488,18 @@ final class AliasDeclaration : BaseNode
     /** */ MemberFunctionAttribute[] memberFunctionAttributes;
 }
 
+///
+final class AliasAssign : BaseNode
+{
+    override void accept(ASTVisitor visitor) const
+    {
+        mixin (visitIfNotNull!(initializer));
+    }
+    mixin OpEquals;
+    /** */ AliasInitializer initializer;
+    /** */ string comment;
```

Comment:
Shouldn't this node contain the identifier of the alias that will be assigned?

Reply:
The `AliasInitializer` contains both the identifier and the initializer.